mathesar/imports/csv.py

```diff
@@ -72,14 +72,16 @@ def get_sv_dialect(file):
 def get_sv_reader(file, header, dialect=None):
     file = TextIOWrapper(file, encoding="utf-8-sig")
     if dialect:
-        reader = csv.DictReader(file, dialect=dialect)
+        reader = csv.DictReader(file, dialect=dialect, quoting=csv.QUOTE_ALL, quotechar='"', delimiter=',')
```

Comment:
Do we need to pass these options here if we're handling quoting at the SQL level?

Reply:
I also don't think we should pass a `delimiter` since this function also handles `TSV` files.

Reply:
When we upload a "bad format" csv with double-quotes, dialect.delimiter cannot return the correct value.

`dialect SimpleDialect(' ', '', '')`